merge_requests.go

```diff
@@ -286,12 +299,21 @@ func (s *MergeRequestsService) UpdateMergeRequest(
 // http://doc.gitlab.com/ce/api/merge_requests.html#accept-mr
 func (s *MergeRequestsService) AcceptMergeRequest(
 	pid interface{},
-	mergeRequest int) (*MergeRequest, *Response, error) {
+	mergeRequest int,
+	opt *AcceptMergeRequestOptions) (*MergeRequest, *Response, error) {
 	project, err := parseID(pid)
 	if err != nil {
 		return nil, nil, err
 	}
-	u := fmt.Sprintf("projects/%s/merge_request/%d/merge", url.QueryEscape(project), mergeRequest)
+
+	v, err := query.Values(opt)
+
+	if err != nil {
+		return nil, nil, err
+	}
+
+	//For some reason putting struct to body doesn't work
+	u := fmt.Sprintf("projects/%s/merge_request/%d/merge?%s", url.QueryEscape(project), mergeRequest, v.Encode())
```

Comment:
I'm testing in on 8.11.6. Also I don't userstand why putting it in body would work in the first place - Gitlab docs say that parameters should be put into URL.